Example/RxRealm/AppDelegate.swift

```diff
@@ -1,6 +1,5 @@
-
-import UIKit
 import RealmSwift
+import UIKit
```

Comment:
Why did you swap those? Not a huge deal but usually it's system-frameworks-first and then 3rd party.